cli/src/pcluster/validators/directory_service_validators.py

```diff
@@ -68,6 +70,31 @@ def _validate(self, domain_name):
             )
 
 
+class PasswordSecretArnValidator(Validator):
+    """PasswordSecretArn validator."""
+
+    def _validate(self, password_secret_arn):
+        """Validate that PasswordSecretArn contains an ARN of a readable secret in AWS Secrets Manager."""
+        try:
+            # We only require the secret to exist; we do not validate its content.
+            AWSApi.instance().secretsmanager.describe_secret(password_secret_arn)
+        except AWSClientError as e:
+            if e.error_code == "ResourceNotFoundExceptionSecrets":
+                self._add_failure("The secrets provided in PasswordSecretArn does not exist.", FailureLevel.ERROR)
+            elif e.error_code == "AccessDeniedException":
+                self._add_failure(
+                    "Cannot validate PasswordSecretArn due to lack of permissions. "
```

Comment:
Same here:

`Cannot validate Secret Arn {secret_arn} due to lack of permissions`

Reply:
Will fix in the next rev, thanks